benchmarks/trace/grpc/benchmark_test.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2017, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package benchmarks
+
+import (
+	"net"
+	"sync"
+	"testing"
+	"time"
+
+	"golang.org/x/net/context"
+
+	"go.opencensus.io/trace"
+	"google.golang.org/grpc"
+)
+
+func untracedPong(ctx context.Context, client PingerClient) (*Pong, error) {
```

Comment:
You are benchmarking your own instrumentation here, you should benchmark the grpctrace package instead.

Reply:
Thanks. Let me fix that.